Sources/Sentry/include/HybridPublic/PrivateSentrySDKOnly.h

```diff
@@ -156,8 +157,16 @@ typedef void (^SentryOnAppStartMeasurementAvailable)(
  * configurations even when targeting iOS or tvOS platforms.
  */
 + (NSData *)captureViewHierarchy;
+
 #endif // SENTRY_UIKIT_AVAILABLE
 
+#if SENTRY_HAS_UIKIT && !TARGET_OS_VISION
+/**
+ * This is how hybrid SDKs can control session replay.
+ */
++ (nullable SentrySessionReplayIntegration *)createReplayIntegration;
+#endif
```

Comment:
How should we use this in hybrid SDKs? There's already an instance installed among integrations by the time the SDK starts. 

Do you suggest removing the original integration from options during `init()`, like so:
```objective-c
        options.integrations = [options.integrations filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(NSString * integration, NSDictionary *bindings) {
            return ![integration isEqualToString:@"SentrySessionReplayIntegration"];
        }]];
```

and then adding a custom hybrid SDK replay integration that holds an instance of `SentrySessionReplayIntegration` (created by this function) as a property and delegates to it? Or did you have another scenario in mind?